dev/freestanding/femr/config/keycloak/realm-data.json

```diff
@@ -2612,6 +2612,7 @@
     "users": [
       {
         "id": "794d2782-e783-4cbb-ae72-b165a4c06e7f",
+        "attributes": {"DEA": "testDEAvalue"},
```

Comment:
Is `testDEAvalue` just the default value if we don't modify a user to add a DEA number?

Would it make sense to omit this line, so we can safely assume if a user has this attribute, it's been added by a human (and correct)?

Reply:
well, this is only set on the `test` account - others generated won't get this default.  i think it's effective for testing at this time.